products/redhat.md

```diff
@@ -11,32 +11,69 @@ releasePolicyLink: https://access.redhat.com/support/policy/updates/errata
 activeSupportColumn: true
 releaseDateColumn: true
 releaseColumn: false
-sortReleasesBy: 'release'
+sortReleasesBy: 'cycleShortHand'
 releaseLabel: "RHEL __RELEASE_CYCLE__"
+LTSLabel: "<abbr title='Extended Life Cycle Support'>ELS</abbr>"
 releases:
   - releaseCycle: "9"
     release: 2022-05-18
     support: 2027-05-31
     eol: 2032-05-31
     latest: "9.0"
+    cycleShortHand: 10
```

Comment:
this one still looks doesnt right , could you turn all cycleshorthand values to double
digits ( like 7 -> 70  , 8 -> 80 ) so we can give ELS version's cycleshorthand values like 81 , 91 , .. 
that does solve possible misunderstandings

Reply:
Fixed with https://github.com/endoflife-date/endoflife.date/pull/1254/commits/4949afbed12be184be16933752d419624c0a9b77